frontend/ui/widget/infomessage.lua

```diff
@@ -5,6 +5,7 @@ It vanishes on key press or after a given timeout.
 
 Example:
     local UIManager = require("ui/uimanager")
+    local InfoMessage = require("ui/widget/infomessage")
```

Comment:
Coding style

Reply:
Ah, true, didn't notice they were out of order ;).